test/Libraries/Microsoft.Extensions.Telemetry.Tests/Logging/ExtendedLoggerTests.cs

```diff
@@ -119,6 +122,58 @@ public static void FeatureEnablement(bool enableRedaction, bool enableEnrichment
         }
     }
 
+#if NET9_0_OR_GREATER
+    [Fact]
+    public static void GlobalBuffering()
```

Comment:
Similar to Http, it would be nice to have an example that uses global buffering in some canonical way that we expect end-users to use it.

Reply:
Added GlobalBuffering_CanonicalUsecase